### DISCUSSION TITLE
theme-live-codeblock - swap placement of editor and preview panels

### DISCUSSION THREAD

**Discussion by binhxn** [facebook/docusaurus#4564] [Q&A] [answered]
Hello,

Currently, the interactive live editor displays the live editor panel above the preview panel. I was wondering if it's possible to swap the placement of the live editor and preview panels so that the preview panel is above the live editor panel? I noticed with the raw implementation of react-live, you can easily swap the placement of the `<LiveEditor />` and `<LivePreview />` components, but couldn't find anything in the Docusaurus documentation.

Any ideas on this? Thank you in advance.

**Comment by slorber** [comment, marked as answer]
`playgroundPosition: "top"` ? :)

https://docusaurus.io/docs/api/themes/@docusaurus/theme-live-codeblock#configuration 

Otherwise you'd have to use swizzle

**Reply by binhxn** [reply]
@slorber Oh my goodness, that was silly of me 🤦 . I was only looking through the Docs/Guides section instead of the Theme Configuration page. Thank you for your help!
